x-pack/solutions/observability/plugins/synthetics/server/feature.ts

```diff
@@ -77,6 +77,27 @@ const canManagePrivateLocationsPrivilege: SubFeaturePrivilegeGroupConfig = {
   ],
 };
 
+const canReadParamsPrivilege: SubFeaturePrivilegeGroupConfig = {
+  groupType: 'independent',
+  privileges: [
+    {
+      id: 'can_read_param_values',
+      name: i18n.translate('xpack.synthetics.features.canReadParams.label', {
+        defaultMessage: 'Can Read Global Parameter values',
```

Comment:
I feel like the style guidance is sentence case? Maybe double check with @nastasha-solomon 
```suggestion
        defaultMessage: 'Can read global parameter values',
```